R/mle-finder.R

```diff
@@ -0,0 +1,23 @@
+MLE_pseudo_inverse <- function(design, outcome) {
+  return(solve(t(design) %*% design, t(design) %*% outcome))
+}
+
+linear_loglik <- function(coef_estimate, design, outcome, noise_var = 1) {
```

Comment:
I'd even remove the `_estimate` from the `coef_estimate`, since it doesn't have to be an estimate, just some coefficients we want the log likelihood for.